full-stack-tests/backend/package.json

```diff
@@ -40,7 +40,7 @@
     "@bentley/cif-subsurface-schema": "^1.0.3",
     "@bentley/cif-subsurface-conflict-analysis-schema": "^1.0.3",
     "@bentley/cif-units-schema": "^1.0.1",
-    "@bentley/imodeljs-native": "3.5.6",
+    "@bentley/imodeljs-native": "3.5.10",
```

Comment:
i'll make sure to remove this in a separate pr; we dont need a direct dep on this

Reply:
dont want to make you push another commit and have to run the pipelines all over again

Reply:
thx!